File: JavaPatternPrograms/SymbolPatterns/Pattern 79.java
class Pattern_229
{
    public static void main(String args[])
    {
        int n = 6; //size
        for (int i = 1; i <= n; i++)
        {
            for (int j = 1; j <= n; j++)
            {
                if (i % 2 == 0) // for even rows
                {
                    if (j % 2 != 0)
                    {
                        System.out.print("x");
                    }
                    else
                    {
                        System.out.print("o");
                    }
                }
                else // for odd rows
                {
                    if (j % 2 != 0)
                    {
                        System.out.print("o");
                    }
                    else
                    {
                        System.out.print("x");
                    }
                }
            }
            System.out.println("");
        }

    }
}

File: PythonPatternPrograms/Symbol Patterns/Pattern 79.py
n=6
for x in range(1, n+1):
    for y in range(1, n+1):
        if x % 2 == 0: # for even rows
            if y % 2 != 0:
                print("x",end=" ")
            else:
                print("o",end=" ")
        else: # for odd rows
            if y % 2 != 0:
                print("o",end=" ")
            else:
                print("x",end=" ")
    print()